python_bindings/dda_model/__init__.py

```diff
@@ -190,9 +190,11 @@ def gradients(
     @property
     def parameters(self):
         return self._model.getParameters()
+    
+    def allParameters(self):
+        return self._model.getDielectrics()
 
     @parameters.setter
     def parameters(self, value):
         # TODO: Shape / type conversion from input array.
-        filter_max_iteration = 1
-        self._model.setParameters(value, filter_max_iteration)
+        self._model.setParameters(value)
```

Comment:
Good catch. Thanks for the fix. Also - fixing this should've been its own commit (rather than committing 100s of lines of code and a half dozen files at once).

Reply:
that makes sense - I can maybe add a commit to the main branch with just this change, along with maybe the binarizers and optimizers modules (keeping the calculations separate)